### ISSUE TITLE
｢method｣ declarator not is searchable

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/doc#671]
This issue is part of https://github.com/perl6/doc/issues/657

```
Syntax features:
    * dispatch related declarators:  multi, only, proto, method, etc...
```

There's probably enough info about methods, but “method” is not searchable as a declarator. For example, “multi” is searchable, so “method” should be searchable as well.

**Comment by gfldex** [comment]
The method declarator is not searchable because it wasn't written yet. Eventually it will appear all by itself under /typesystem#method because there will be a heading with that name.

**Comment by gfldex** [comment]
fixed with a38088abed
